### PR TITLE
Feature Alerts -  Successful and Error

### PR DESCRIPTION
**Pull request by julianguerrerop202**
image
image

### CODE REVIEW

**Review by brsmallp202 — APPROVED** [review]
Everything looks good, thank you @julianguerrerop202

File: projects/ngx-aus-design-system/src/lib/blocks/components/alert/alert.module.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { HttpClientModule } from "@angular/common/http";
+import { SanitizePipeModule } from "../../../shared/pipes/sanitize/sanitize.module";
+import { TypographyModule } from "../../../shared/components/typography/typography.module";
+
+@NgModule({
+  declarations: [AlertComponent],
+  imports: [
+    ArrowLinkModule,
+    CommonModule,
+    CookieModule.withOptions(),
```
Comment (andresliu-p202) [inline projects/ngx-aus-design-system/src/lib/blocks/components/alert/alert.module.ts:16]: Do we need `ArrowLink` and `Cookie` modules for this component?

File: projects/ngx-aus-design-system/src/lib/blocks/components/alert/alert.stories.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import { SanitizePipe } from "../../../shared/pipes/sanitize/sanitize.pipe";
+import { Array2StringPipe } from "../../../shared/pipes/array2string/array2string.pipe";
+import { WysiwygComponent } from "NgxAusDesignSystem";
+
+export default {
+  decorators: [
+    moduleMetadata({
+      declarations: [
+        TypographyComponent,
+        WysiwygComponent,
```
Comment (andresliu-p202) [inline projects/ngx-aus-design-system/src/lib/blocks/components/alert/alert.stories.ts:15]: Also here

File: projects/ngx-aus-design-system/src/lib/blocks/components/alerts/alerts.module.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  ],
+  imports: [
+    AlertModule,
+    ArrowLinkModule,
+    CommonModule,
+    CookieModule.withOptions(),
+    HttpClientModule,
+    SanitizePipeModule,
+    TypographyModule,
+    WysiwygModule,
```
Comment (andresliu-p202) [inline projects/ngx-aus-design-system/src/lib/blocks/components/alerts/alerts.module.ts:24]: Same modules here

**Review by andresliu-p202 — APPROVED** [review]

File: projects/ngx-aus-design-system/src/lib/blocks/components/alerts/alerts.stories.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import { Story, Meta, moduleMetadata, componentWrapperDecorator } from "@storybook/angular";
+import { LazyLoadImageModule } from "ng-lazyload-image";
+import { CommonModule } from "@angular/common";
+
+import { TypographyComponent } from "../../../shared/components/typography/typography.component";
+import { AlertsComponent } from "./alerts.component";
+import { SanitizePipe } from "../../../shared/pipes/sanitize/sanitize.pipe";
+import { Array2StringPipe } from "../../../shared/pipes/array2string/array2string.pipe";
+import { WysiwygComponent } from "NgxAusDesignSystem";
```
[the remark this answers is not in the document]
Comment (julianguerrerop202) [reply projects/ngx-aus-design-system/src/lib/blocks/components/alerts/alerts.stories.ts:9]: Yes, I tested that, if we remove any of those, the alert will have an error.

File: projects/ngx-aus-design-system/src/lib/blocks/components/alerts/alerts.module.ts (hunk 3 above)
Comment (brsmallp202) [reply projects/ngx-aus-design-system/src/lib/blocks/components/alerts/alerts.module.ts:24]: Do we have to import these for the module file in any other components?

File: projects/ngx-aus-design-system/src/lib/blocks/components/alert/alert.stories.ts (hunk 2 above)
Comment (julianguerrerop202) [reply projects/ngx-aus-design-system/src/lib/blocks/components/alert/alert.stories.ts]: Yes, thank you, I removed many of those, please check again

File: projects/ngx-aus-design-system/src/lib/blocks/components/alert/alert.module.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    HttpClientModule,
+    SanitizePipeModule,
+    TypographyModule,
+    WysiwygModule,
```
[the remark this answers is not in the document]
Comment (julianguerrerop202) [reply projects/ngx-aus-design-system/src/lib/blocks/components/alert/alert.module.ts]: Yes, thank you, I removed many of those, please check again

File: projects/ngx-aus-design-system/src/lib/blocks/components/alerts/alerts.module.ts (hunk 3 above)
Comment (julianguerrerop202) [reply projects/ngx-aus-design-system/src/lib/blocks/components/alerts/alerts.module.ts]: Yes, thank you, I removed many of those, please check again

**Review by brsmallp202 — APPROVED** [review]

**Pull request merged**